### DISCUSSION TITLE
Signin oauth with github redirects to localhost

### DISCUSSION THREAD

**Discussion by jhcueva** [supabase/supabase#12211] [Questions]
Hello everyone,

I'm using supabase signInWithOAuth to sign in using github, all is working great while working on local. However, I've deployed my app and when I try to sign in, enter my email and password but at the end redirects me to localhost in stead of the url of the app.
Does anyone knows how to redirect the authentication to my app url?

Code use to authenticate
```
await supabase.auth.signInWithOAuth({ provider: "github" },)
```

**Comment by j4w8n** [comment]
Did you add your public url to the supabase dashboard settings?
